draft-ietf-capport-api.md

```diff
@@ -130,7 +130,11 @@ Upon receiving this information the client will provide this information to the
 
 # Security Considerations {#security}
 
-TBD: Provide complete security requirements and analysis.
+One of the goals of this protocol is to improve the security of the communication between client hosts and Captive Portal systems. Client traffic is protected from passive listeners on the local network by requiring TLS-encrypted connections between the client and the Captive Portal API server, as described in {{api-details}}. All communication between the clients and the API server MUST be encrypted.
+
+In addition to ecrypting communications between clients and Captive Portal systems, thie protocol requires a basic level of authentication from the API server, as described in {{server-auth}}. Specifically, the API server MUST present a valid certificate on which the client can perform revocation checks. This allows the client to ensure that the API server has authority for a hostname that can be presented to a user. This hostname MUST be presented to the user when associating with the Captive Portal, in order to benefit from the authentication check.
```

Comment:
Two minor typos:
```suggestion
In addition to encrypting communications between clients and Captive Portal systems, this protocol requires a basic level of authentication from the API server, as described in {{server-auth}}. Specifically, the API server MUST present a valid certificate on which the client can perform revocation checks. This allows the client to ensure that the API server has authority for a hostname that can be presented to a user. This hostname MUST be presented to the user when associating with the Captive Portal, in order to benefit from the authentication check.
```

I don't know about the last "MUST" here.  The name comes from an unauthenticated source, so the value in presenting that information is not that great.  That doesn't mean that it couldn't be used, but to require that it be is probably going too far.

Reply:
Fixed the typos! Also, removed the sentence with the MUST.